cardano-config/cardano-config.cabal

```diff
@@ -42,14 +45,17 @@ library
                      , optparse-applicative
                      , ouroboros-consensus
                      , ouroboros-network
+                     , process
                      , iproute
                      , safe-exceptions
                      , scientific
                      , string-conv
                      , stm
+                     , template-haskell
                      , text
                      , transformers
                      , transformers-except
+                     , unordered-containers
```

Comment:
why is this new dependency needed?

Reply:
Not needed anymore, good point, dropped.